pyVMC.py

```diff
@@ -1607,6 +1607,57 @@ def newBGPprefixlist(csp_url, session_token):
         else:
             print("Please choose 1, 2, 3 or 4 - Try again or check the help.")
 
+def attachT0BGPprefixlist(csp_url, session_token, prefix_list_id, route_filter_dir, neighbor_id):
+    myHeader = {'Authorization': f'Bearer {session_token}', 'Content-type': 'application/json'}
+    myURL = f'{csp_url}/policy/api/v1/infra/tier-0s/vmc/locale-services/default/bgp/neighbors/' + neighbor_id
+    response = requests.get(myURL, headers=myHeader)
+    direction = route_filter_dir + '_route_filters'
+    if response.status_code == 200:
+        json_response = response.json()
+        neighbor_json = json_response
+        del neighbor_json['_create_time']
+        del neighbor_json['_create_user']
+        del neighbor_json['_last_modified_time']
+        del neighbor_json['_last_modified_user']
+        del neighbor_json['_system_owned']
+        del neighbor_json['_protection']
+        del neighbor_json['_revision']
```

Comment:
When hidden properties are skipped in this manner, it introduces future bugs when an additional hidden property gets added. Consider looping with a for loop and deleting properties that start with '_'